src/qt/bitcoingui.h

```diff
@@ -90,12 +90,14 @@ class BitcoinGUI : public QMainWindow
     QAction *historyAction;
     QAction *quitAction;
     QAction *sendCoinsAction;
+    QAction *sendCoinsMenuAction;
```

Comment:
Yes, it's mostly to satisfy static checkers, as well as a belt-and-suspenders measure in general. Thanks.